rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -552,6 +556,133 @@
           }
         }
       }
+    },
+    "url": {
+      "type": "string",
+      "title": "URL",
+      "placeholder": "Example: http://server.mediatheque.ch/CH-000019-X:223094.file",
+      "format": "uri",
+      "pattern": "^(ftp?|http?)://.*$",
+      "minLength": 7,
+      "form": {
+        "validation": {
+          "messages": {
+            "patternMessage": "Should be a valid URL format."
+          }
+        },
+        "templateOptions": {
+          "cssClass": "editor-title"
+        },
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
+    },
+    "pac_code": {
+      "title": "Conservation and preservation policy",
+      "type": "string",
+      "enum": [
+        "0_frozen_collection",
+        "1_reserved",
+        "2_controlled_consumption",
+        "3_consumables",
+        "4_disposable"
+      ],
+      "form": {
+        "type": "selectWithSort",
+        "options": [
+          {
+            "label": "4_disposable",
+            "value": "4_disposable"
+          },
+          {
+            "label": "3_consumables",
+            "value": "3_consumables"
+          },
+          {
+            "label": "2_controlled_consumption",
+            "value": "2_controlled_consumption"
+          },
+          {
+            "label": "1_reserved",
+            "value": "1_reserved"
+          },
+          {
+            "label": "0_frozen_collection",
+            "value": "0_frozen_collection"
+          }
+        ],
+        "templateOptions": {
+          "cssClass": "editor-title"
+        },
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
+    },
+    "price": {
+      "title": "Item price",
+      "type": "number",
+      "minimum": 0,
+      "form": {
+        "templateOptions": {
+          "cssClass": "editor-title"
+        },
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
+    },
+    "legacy_checkout_count": {
+      "title": "Virtua checkout count for item",
+      "type": "integer",
+      "minimum": 0,
+      "default": 0
+    },
+    "legacy_circulation_rules": {
+      "title": "Item circulation rules",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "use_item_specific_rules": {
+          "title": "To use item specific rules or CIPO rules",
+          "type": "boolean",
+          "default": false
+        },
+        "loan_duration": {
+          "title": "Loan duration in days",
+          "type": "integer",
+          "minimum": 0,
+          "default": 0
+        },
+        "allow_request": {
+          "title": "Allow requests",
+          "type": "boolean",
+          "default": true
+        },
+        "floats": {
+          "title": "Item floats",
+          "type": "boolean",
+          "default": false
+        }
+      }
+    },
+    "_masked": {
+      "title": "Masked",
```

Comment:
Add description: "A masked resource is visible in the professional interface, but not in the public interface."